### ISSUE TITLE
batchNorm or layerNorm？

### ISSUE THREAD

**Issue by Napier7** [lucidrains/segformer-pytorch#10]
I see the author using batchNorm not layerNorm according to the mmsegmentation config file in the official depot. Am I misinterpreting this？
